ydb/core/fq/libs/row_dispatcher/format_handler/ut/common/ut_common.h

```diff
@@ -42,6 +42,11 @@ class TBaseFixture : public NUnitTest::TBaseFixture, public TTypeParser {
         TBatch() = default;
         TBatch(std::initializer_list<TRow> rows);
 
+        template <typename TIterator>
+        explicit TBatch(const TIterator& begin, const TIterator& end)
```

Comment:
Кажется тут explicit лишний (есть два аргумента в конструкторе)


```suggestion
        TBatch(const TIterator& begin, const TIterator& end)
```